### PR TITLE
completed pet store contact

### PR DESCRIPTION
**Pull request by vandickersonjr**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+	<!-- end of nav -->
+  <div class="container">
+    <div class="row">
+      <div class="col-md-7">
+        <!-- veterinarians -->
+        <div class="row">
+        <!-- joe -->
+          <div class="col-md-4">
+            <img src="img\doctor1.jpg" alt="Joe Smith" class="img-responsive img-rounded shadow">
```
Comment (rockygg) [inline contact.html:38]: does this img work? it's typically written as `img/doctor1.jpg`

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+          </div>
+          <div class="col-md-8">
+            <h3>Dr. Joe Smith</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt delectus molestias quo fugiat obcaecati consectetur ut sint incidunt, dignissimos minima facilis nostrum aliquam officiis! Explicabo esse autem veniam excepturi quia?</p>
+          </div>
+        </div>
+        <!-- jane -->
+        <div class="row">
+          <div class="col-md-4">
+            <img src="img\doctor2.jpg" alt="Jane Smith" class="img-responsive img-rounded shadow">
```
Comment (rockygg) [inline contact.html:48]: same question as above ^

**Review by rockygg — APPROVED** [review]
Good job, I would just double check all your images, best practice is to use the forward slash in the file paths

File: contact.html (hunk 1 above)
Comment (vandickersonjr) [reply contact.html:38]: Yes, I copied the file path. That is how it pasted to my code.

**Pull request merged**